include/verilated_sym_props.h

```diff
@@ -94,45 +87,18 @@ class VerilatedVarProps VL_NOT_FINAL {
         , m_vlflags{vlflags}
         , m_pdims{pdims}
         , m_udims{udims} {
-        initUnpacked(nullptr);
+        for (auto i = 0; i < m_udims; ++i) {
+            m_unpacked.emplace_back(0, 0);
+        }
+        for (auto i = 0; i < m_pdims; ++i) {
+            m_packed.emplace_back(0, 0);
```

Comment:
Are you dropping left/right or is that done elsewhere?  Check that you validate left/right in tests, thanks.

Reply:
the deleted function initUnpacked() was only ever called with a null pointer. The function varInsert() in verilated.cpp fills the left/right values after construction

Reply:
ok, add a comment there noting that, thanks.

Reply:
Looks like that function was used by generated files. I modified it and commented